activate

```diff
@@ -71,25 +82,45 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 # detect live system
 live="false"
 if grep -qw "endless.live" /proc/cmdline; then
-    image="true"
+    live="true"
+fi
+
+
+# We don't want dummy serials sent
+if [ "${serial}" == "To be filled by O.E.M." ]; then
```

Comment:
We already have these kind of bogus entries in the metrics system from OEMs who have missed completing things - in all three fields from DMI we can get "To Be Filled By O.E.M." (actually more popular) or "To be filled by O.E.M.", and one of our 6th most popular make/models is the award-winning "System Product Name" from "System manufacturer". What I'm saying is bogus information here is a fact of life and I think trying to sanitise it is a losing battle. I don't think we want to get into trying to filtering on the client - we can just deal with this on the server by permitting these to be non-unique and assuming every activation is a potentially relevant event and storing it anyway - then if we do need to analyse them and discard bogus stuff or de-duplicate we can do it on the aggregate data-set.